tensorflow/lite/micro/tools/ci_build/test_project_generation.sh

```diff
@@ -87,3 +87,33 @@ PATH=${PATH}:${ROOT_DIR}/tensorflow/lite/micro/tools/make/downloads/gcc_embedded
 popd > /dev/null
 
 #rm -rf ${TEST_OUTPUT_DIR_CMSIS}
+
+# Test that C++ files are renamed to .cpp
+rm -rf ${TEST_OUTPUT_DIR_CMSIS}
```

Comment:
the commented out like 89 seems to be an oversight. Let's move this rm to the CMSIS block and add a new output dir that tests the file renaming.

```
#rm -rf ${TEST_OUTPUT_DIR_CMSIS}

# Test that C++ files are renamed to .cpp
rm -rf ${TEST_OUTPUT_DIR_CMSIS}
```

should become:
```
rm -rf ${TEST_OUTPUT_DIR_CMSIS}

# Test that C++ files can be renamed to .cpp
```

Reply:
The above diff isn't showing the change but I implemented it as you said with the rm for the CMSIS directory being uncommented and then leading into the new test block for the renaming feature.

Can be seen in the latest commit from here: https://github.com/tensorflow/tflite-micro/commit/768fb9f7997d1fddf8c3bdadd48acf5e4c0225b5#diff-227b53ea64f9bfbf4262fdcbd7fee872cc32deb19774f0a947850db6828080deR89